src/nvim/os/env.c

```diff
@@ -28,11 +28,20 @@
 #endif
 
 const char *os_getenv(const char *name)
```

Comment:
This should be documented now that the behavior differs from `getenv`.
I think a reference such as `@see {os_unsetenv}` would be useful.